pages/components/Day.tsx

```diff
@@ -0,0 +1,72 @@
+import { Component } from "react";
+import ShowDailySchedule from "./ShowDailySchedule";
+import DayButton from './DayButton';
+
+interface IProps {
+  currDay: string;
+  slots: {};
+}
+class Day extends Component<IProps, any> {
+  constructor(props) {
+    super(props);
+    this.state = {
+      clicked: false
+    };
+  }
+
+  showSchedule = () => {
+    if (this.state.clicked) {
+      this.setState({
+        clicked: false
+      });
+    } else {
+      this.setState({
+        clicked: true
+      });
+    }
+  };
+
+  render() {
+    return (
+      <div>
+        {this.state.clicked && (
```

Comment:
this and the other seems to be the excact same? is it necessary to create two buttons when you set clicked prop in the component?